### PR TITLE
Improve language server error message

### PR DESCRIPTION
**Pull request by ishitatsuyuki**

### CODE REVIEW

File: packages/languages/src/browser/language-client-contribution.ts [hunk 1]
```diff
@@ -98,8 +98,9 @@ export abstract class BaseLanguageClientContribution implements LanguageClientCo
             commands: this,
             documentSelector: this.documentSelector,
             synchronize: { fileEvents },
-            initializationFailedHandler: () => {
-                this.messageService.error("Failed to start language server '" + this.name + "'.");
+            initializationFailedHandler: (err) => {
+                let detail = err instanceof Error ? `: ${err.message}` : '.';
```
Comment (kittaakos) [inline packages/languages/src/browser/language-client-contribution.ts]: Please change `let` to `const`.

File: packages/languages/src/browser/language-client-contribution.ts [hunk 2]
```diff
@@ -98,8 +98,9 @@ export abstract class BaseLanguageClientContribution implements LanguageClientCo
             commands: this,
             documentSelector: this.documentSelector,
             synchronize: { fileEvents },
-            initializationFailedHandler: () => {
-                this.messageService.error("Failed to start language server '" + this.name + "'.");
+            initializationFailedHandler: (err) => {
```
Comment (kittaakos) [inline packages/languages/src/browser/language-client-contribution.ts]: Please remove the parentheses around `err`

File: packages/languages/src/browser/language-client-contribution.ts (hunk 1 above)
Comment (akosyakov) [reply packages/languages/src/browser/language-client-contribution.ts]: thought we have linting errors for it, surprisingly build does not fail

Comment (kittaakos) [reply packages/languages/src/browser/language-client-contribution.ts]: Oh well, good point. Does it worth a follow-up task?

Comment (akosyakov) [reply packages/languages/src/browser/language-client-contribution.ts]: yes, would be nice to fix it

**Comment by kittaakos** [comment]
I have restarted the AppVeyor build.

**Pull request merged**